app/models/guest.rb

```diff
@@ -0,0 +1,11 @@
+class Guest < ApplicationRecord
+  has_many :room_guests
+  has_many :rooms, through: :room_guests
+
+  validates_presence_of :name, :nights
+  validates_numericality_of :nights
```

Comment:
Good job using validations here, we won't ask you to do these on the final though they are optional for the ICs.